analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -549,6 +557,13 @@ def main(args):
     if 'none' in args.compile_uniqueing and 'ctu_phases' in args:
         args.compile_uniqueing = "alpha"
 
+    compiler_info_file = None
+    if 'compiler_info_file' in args:
+        LOG.debug("Compiler info is read from: %s", args.compiler_info_file)
+        compiler_info_file = args.compiler_info_file
```

Comment:
Should we check if the value of `args.compiler_info_file` is not `None`? If it is set on the command line without a value what will be the value here?

Reply:
if compiler_info_file is in args then it cannot be None.
If no parameter is given to --compiler_info_file the argparser complains that --compiler-info-file must have 1 argument.

I extended this with the verification wether args.compiler_info_file exists.